experimental/examples/ordered-list/src/orderedlist.ts

```diff
@@ -0,0 +1,166 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { DataObject, DataObjectFactory } from "@fluidframework/aqueduct";
+import { IEvent } from "@fluidframework/common-definitions";
+import {
+    ChangeNode,
+    Definition,
+    Delete,
+    Insert,
+    Move,
+    NodeId,
+    SharedTree,
+    StablePlace,
+    StableRange,
+    TraitLabel,
+} from "@fluid-experimental/tree";
+import { v4 as uuid } from "uuid";
+import { Serializable } from "@fluidframework/datastore-definitions";
+import { IFluidHandle } from "@fluidframework/core-interfaces";
+
+const listItemDefinition = undefined as unknown as Definition;
+const itemsTrait = "" as TraitLabel;
+
+const makePositionNode = <T>(value: Serializable<T>): ChangeNode => ({
+    identifier: uuid() as NodeId,
```

Comment:
how does it ensure everybody ends up with the  same list when there are concurrent insertions to the same position?

Reply:
The Fluid service assigns an increasing seq# to each op as it is received before broadcasting them to clients. This seq# defines a total ordering that Fluid DDSes must honor when constructing the client's local copy of the data.

The only "unordered" ops that a DDS must deal with are your client's local edits that have not yet been sequenced by the Fluid service. However, the DDS knows that all of these unordered local edits will be assigned higher sequence numbers than the remote ops it receives.

Therefore, all copies of a Fluid DDS have the same relative ordering for the ops the DDS is aware of, and consequently two DDSes that are aware of the same ops will produce identical state.